examples/tracing/kernel_cache_alloc_free_example.txt

```diff
@@ -0,0 +1,23 @@
+Demonstrations of kernel_cache_alloc_free.py, the Linux eBPF/bcc version.
+
+This tracks kmem_cache_alloc and kmem_cache_free routines (using tracepoints).
+
+
+# ./kernel_cache_alloc_free
+Kernel cache allocation started ... Hit Ctrl-C to end!
+TIME(S)            Call_site              ptr                      bytes_req       bytes_alloc  gfp_flags                      Command                       
+8308825716036      18446744072166397952   0x18446629912970690560   4096            4096         0x7882827029131821248          _cache_alloc                  
```

Comment:
This output is incorrect. `_cache_alloc` is printed out instead of `kmem_cache_alloc`. See my previous comments. Please update the output format to be similar to kernel tracepoint output.